core/mock_test.go

```diff
@@ -181,6 +181,23 @@ func (m mockBackend) BuildRoundChangeMessage(
 	}
 }
 
+func (m mockBackend) HasQuorum(view *proto.View, messages []*proto.Message) bool {
```

Comment:
Could we use height and messages int instead of view and *proto.messages?

Reply:
Good one, I'd say yes for the current implementation, but I guess we'll need the `From` field from the `Message` to compute voting power.

Reply:
Could we then have 1 narrow interfaces instead and then extend them is needed?

Reply:
You mean encapsulating params in an interface? can you give an example?